web/src/components/time-range-select-modal.css

```
@@ -1,18 +1,20 @@
 .lv-plugin__time-range-modal {
+  --pf-t--global--font--size--md: 14px;
   --pf-v5-global--FontSize--md: 14px;
+  width: fit-content !important;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)